CV-Portfolio/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">

```

Comment:
Use `<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">` here.
Setting `width=device-width` ensures that the viewport width matches the actual width of the device screen. Without this, the website would default to a standard width, typically 980px, causing content to appear too small on mobile screens.